ocaml/xapi/xapi_vm_lifecycle.ml

```diff
@@ -514,6 +522,12 @@ let check_operation_error ~__context ~ref ~op ~strict =
       then check_vgpu ~__context ~op ~ref_str ~vgpus:vmr.Db_actions.vM_VGPUs
       else None) in
 
+  (* The VM has a VUSB, check if the operation is allowed*)
+  let current_error = check current_error (fun () ->
+      if vmr.Db_actions.vM_VUSBs <> []
```

Comment:
The check `vmr.Db_actions.vM_VUSBs <> []` is repeated in `check_vusb`. I think the code from above should be simply inlined here unless we need `check_vusb` somewhere else.